parser.y

```diff
@@ -2137,7 +2137,7 @@ ColumnDef:
 			yylex.AppendError(yylex.Errorf("Invalid column definition"))
 			return 1
 		}
-        $$ = colDef
+        	$$ = colDef
```

Comment:
could you replace all the spaces to tab?

Reply:
Done.